README.adoc

```diff
@@ -122,23 +93,27 @@ Now, navigate back to the `start` directory to begin.
 
 
 // =================================================================================================
-// Enabling fault tolerance in the application
+// Enabling fault tolerance
 // =================================================================================================
 
-== Enabling fault tolerance in the application
+== Enabling fault tolerance
+
+The MicroProfile Fault Tolerance API was added as a dependency to your `pom.xml` file. Look for
+the dependency with the `org.eclipse.microprofile.fault-tolerance` group ID.
+Adding this dependency allows you to use the fault tolerance policies in your microservices.
 
-The MicroProfile Fault Tolerance API is added as a dependency to your `pom.xml` file. Look for
-the dependency with the `org.eclipse.microprofile.fault-tolerance` group ID. Use this feature
-to build fault-tolerant microservices with the MicroProfile Fault Tolerance API.
-The `mpFaultTolerance-1.0` feature is also enabled in the `src/main/liberty/config/server.xml`
-file.
+You can also find the `mpFaultTolerance-1.0` feature in your `src/main/liberty/config/server.xml` server configuration,
+which turn on MicroProfile Fault Tolerance in Open Liberty.
```

Comment:
"turn" should be "turns" with the "s".

Also, are we referring to the API? If so, we might want to say, "...turns on the MicroProfile Fault Tolerance API...". Or if we're speaking of fault tolerance in general, we can say, "...turns on fault tolerance in Open Liberty."

Reply:
It enables or turns on the MP Fault Tolerance feature (capabilities) in OL.

Reply:
Fixed